src/Bicep.Core.Samples/Files/InvalidModules_LF/main.bicep

```diff
@@ -406,7 +406,7 @@ module paramNameCompletionsInFilteredLoops 'modulea.bicep' = [for (x,i) in empty
   name: 'hello-${x}'
   params: {
     // #completionTest(0,1,2) -> moduleAParams
-
+  
```

Comment:
Adding the whitespaces because line 409 is empty, and `completionTest(2)` will trigger completions immediately before `}` (`line409\r\n|line410}`). With the changes of the PR it will return an empty list.